src/audio/eq_iir/llext/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2024 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_path(SET SOF_BASE NORMALIZE ${CMAKE_CURRENT_SOURCE_DIR}/../../../..)
+
+set(MODULE "eq_iir")
+
+file(STRINGS ${CMAKE_CURRENT_LIST_DIR}/../${MODULE}.toml uuids REGEX "^[ \t]*uuid *=")
+
+foreach(line IN LISTS uuids)
+	# extract UUID value - drop the 'uuid = ' part of the assignment line
+	string(REGEX REPLACE "^[ \t]*uuid *= \"([0-9A-F\-]*)\"" "\\1" uuid ${line})
+	file(APPEND ${ZEPHYR_BINARY_DIR}/${MODULE}_llext/llext.uuid "${uuid}\n")
+endforeach()
+
+add_llext_target(${MODULE}
+	OUTPUT  ${PROJECT_BINARY_DIR}/${MODULE}_llext/${MODULE}.llext
+	SOURCES ../eq_iir.c
+		../eq_iir_ipc4.c
+		../eq_iir_generic.c
+)
+
+target_include_directories(${MODULE}_llext_lib PRIVATE
+	"${SOF_BASE}/xtos/include"
+	"${SOF_BASE}/src/include"
+	"${SOF_BASE}/tools/rimage/src/include"
+)
+
+sof_append_relative_path_definitions(${MODULE}_llext_lib)
+
+add_llext_command(TARGET ${MODULE}
+	PRE_BUILD
+	COMMAND ${CMAKE_C_COMPILER} -E ${CMAKE_CURRENT_LIST_DIR}/llext.toml.h -P -DREM=
+		-I${SOF_BASE} -I${SOF_BASE}src
+		-imacros ../include/generated/autoconf.h
+		-o rimage_config.toml
```

Comment:
This one does not return anything so it is trivial to make it a shared function in zephyr/CMakeLists.txt